cumulus/parachain-template/runtime/src/lib.rs

```diff
@@ -133,11 +133,11 @@ impl_opaque_keys! {
 
 #[sp_version::runtime_version]
 pub const VERSION: RuntimeVersion = RuntimeVersion {
-	spec_name: create_runtime_str!("subspace-executor"),
-	impl_name: create_runtime_str!("subspace-executor"),
+	spec_name: create_runtime_str!("subspace"),
+	impl_name: create_runtime_str!("subspace"),
```

Comment:
I'm not entirely sure why is this needed. They are different runtimes and can have different spec names. Unless I refactored it incorrectly in that PR, they should continue to work the same exact way as before.

Reply:
I guess there might be some subtle differences between the regular standalone runtime and the combined runtime when working with client, I didn't spend more time digging into it though(I'd love to, but just want to spend more time implementing the new features :D).

Reply:
I don't think there should be any difference. I also noticed there doesn't seem to be documentation on how to run executor, can you add it so I can try it out? Something simple without bash scripts would be preferred.

Reply:
You don't have to run an exeuctor to reproduce the regression, it happens for the farmer. Just run a farmer net locally.

Reply:
Hm... I see the issue, that is confusing indeed, let me see what is going on there first.

Reply:
OK, the issue is deeper and cause by the fact that in `no_std` environment some Substrate macros add info to the link sections, which are then included as global info in WASM and read like here (one of the places): https://github.com/paritytech/substrate/blob/36b8970ef68b2f8d031f0aeadbe47d06a2679332/client/executor/src/wasm_runtime.rs#L364-L369

I need to think and tinker with it a bit, this might require some changes to Substrate itself.

Reply:
This PR doesn't fix introduced issue and we'll still have other issues down the road.

Reply:
Left a comment: https://github.com/paritytech/substrate/issues/7327#issuecomment-1079983110

Working on workaround now based on my understanding of Substrate codebase.